client/etcd/client.go

```diff
@@ -36,7 +40,6 @@ import (
 	etcdheartbeat "github.com/m3db/m3cluster/services/heartbeat/etcd"
 	"github.com/m3db/m3cluster/services/leader"
 	"github.com/m3db/m3x/instrument"
-	"github.com/m3db/m3x/log"
```

Comment:
Seems `goimports` might have removed this package (`github.com/m3db/m3x/log`) in favor or `log`. I think you'll need to switch it back though since everything things will expect the former. (This is one of those magic things VS Code runs for you by default that are usually pretty helpful but can go awry at times.)